data/companies.json

```diff
@@ -336,6 +336,22 @@
         "youtube": "https://www.youtube.com/c/Fusemachines"
       }
     },
+    {
+      "name": "Genese Solution",
+      "establishedAt": 1555006500,
```

Comment:
What's the source for this?

Reply:
> What's the source for this?
Here it is :
![Screenshot from 2023-10-21 10-28-18](https://github.com/Know-Nepal/tech-companies/assets/115408327/325393c2-2966-4098-8589-576c60bcc428)